Description: Ignore emoji pack in super-large emoji rendering
 It is necessary to build against the unpatched rLottie lib.
 .
 Although latest versions of the library already support theme overriding, we
 patch the rLottie calls here for easy backporing to buster.
Bug-Samsung: https://github.com/Samsung/rlottie/pull/252
Forwarded: not-needed
Author: Nicholas Guriev <guriev-ns@ya.ru>
Last-Update: Thu, 12 Aug 2021 22:56:23 +0300

--- a/Telegram/lib_lottie/lottie/details/lottie_frame_provider_direct.cpp
+++ b/Telegram/lib_lottie/lottie/details/lottie_frame_provider_direct.cpp
@@ -58,11 +58,7 @@ bool FrameProviderDirect::load(
 			? replacements->replacements
 			: std::vector<std::pair<std::uint32_t, std::uint32_t>>()));
 #else
-	_animation = rlottie::Animation::loadFromData(
-		string,
-		std::string(),
-		std::string(),
-		false);
+	_animation = rlottie::Animation::loadFromData(std::move(string), "");
 #endif
 	if (!_animation) {
 		return false;
--- a/Telegram/lib_lottie/lottie/lottie_icon.cpp
+++ b/Telegram/lib_lottie/lottie/lottie_icon.cpp
@@ -37,11 +37,7 @@ namespace {
 		false,
 		std::move(list));
 #else
-	auto result = rlottie::Animation::loadFromData(
-		std::move(string),
-		std::string(),
-		std::string(),
-		false);
+	auto result = rlottie::Animation::loadFromData(std::move(string), "");
 #endif
 	return result;
 }
